actionpack/test/dispatch/routing_test.rb

```diff
@@ -4592,3 +4592,44 @@ def test_positional_args_with_format_false
     assert_equal '/en/posts/2014/12/13', archived_posts_path(2014, 12, 13)
   end
 end
+
+class TestErrorsInController < ActionDispatch::IntegrationTest
```

Comment:
I found that when a particular setup of controllers is needed, a new test class is used (~ 26) in this file.

I thought that was a good idea to do so, not to pollute the main test class.